UI/CardViewMigration/CardViewMigration/CardViewMigration/Platforms/WebAssembly/LinkerConfig.xml

```diff
@@ -0,0 +1,10 @@
+<linker>
```

Comment:
The original content of LinkerConfig.xml should be restored

Reply:
Thanks @Youssef1313! Except the <assembly fullname="CardViewMigration" /> should be CardViewMigration not CardViewMigration.Wasm now, right?

Reply:
Yes. That's right